### PR TITLE
fix(questions): dont blow up when whoami errors

### PR DESCRIPTION
**Pull request by trevtrich**
- fixes @travi/javascript-scaffolder#482

### CODE REVIEW

File: src/prompts/questions.js [hunk 1]
```diff
@@ -35,6 +36,13 @@ function authorQuestions({name, email, url}) {
 export async function prompt({npmAccount, author}, ciServices, hosts, visibility, vcs, decisions) {
   const npmConf = npmConfFactory();
 
+  let maybeLoggedInNpmUsername;
+  try {
+    maybeLoggedInNpmUsername = (await execa('npm', ['whoami'])).stdout;
+  } catch (failedExecutionResult) {
+    warn('No logged in user found with `npm whoami`. Login with `npm login` to enable default project scoping.');
```
Comment (travi) [inline src/prompts/questions.js:43]:
```suggestion
    warn('No logged in user found with `npm whoami`. Login with `npm login` to use your npm account name as the package scope default.');
```

i wonder if something along these lines would be more clear?

**Comment by travi** [comment]
LGTM. feel free to merge after fixing the lint issue. thanks for getting this knocked out!

**Comment by trevtrich** [comment]
@travi since this one is your personal repo I won't be able to merge this one 😄 . The lint issue is fixed though.

**Comment by travi** [comment]
Ah, right

**Comment by travi** [comment]
Even more reason to get this migrated

**Pull request merged**